docs/Actions/load-users-sql.md

```diff
@@ -0,0 +1,81 @@
+---
+id: LoadUsersSql
+title: Load Users From SQL
+sidebar_label: Load Users From SQL
+---
+
+An action that allows altering the contextual users which all the subsequent actions use. This action will not alter the context user (logged in user).
```

Comment:
Feel that the clarification is not sufficient between what context.User and context.Users mean. They're both in the context.